### ISSUE TITLE
Master and slave example

### ISSUE THREAD

**Issue by mgonzaga1990** [cetic/helm-postgresql#6]
Do you have an example of master and slave?

**Comment by alexnuttinck** [comment]
@mgonzaga1990 There is currently no feature for replication in this Helm Chart.

**Comment by mgonzaga1990** [comment]
@alexnuttinck do we have plan to work on this in the future?

**Comment by alexnuttinck** [comment]
@mgonzaga1990 it is not at the agenda. If someone proposes a PR, I can take the time to review it.
If you need this feature asap, I suggest you to have a look at the [bitnami Helm Chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql).

**Comment by mgonzaga1990** [comment]
yeah..  seen that. the only problem is it is not using the postgre image
from docker as a result it isn't compatible with arm device
